build/ci/build.yml

```diff
@@ -1,122 +1,80 @@
   
-  tools:                                                    # a list of additional .NET global tools needed
+  tools:                                                            # Additional .NET global tools to install
   - 'xamarin.androidbinderator.tool': '0.5.7'
   - 'Cake.Tool': '4.0.0'
   - 'boots': '1.1.0.712-preview2'
   - 'private-api-tools': '1.0.1'
-
-  # Build Parameters
-  verbosity: 'normal'                                       # the build verbosity: 'minimal', 'normal', 'diagnostic'
-  configuration: 'Release'                                  # the build configuration: 'Debug', 'Release'
   
   # Reporting/Analysis Parameters
-  areaPath: 'DevDiv\VS Client - Runtime SDKs\Android'       # the areaPath to log any issues
-  publishJob: ''                                            # the job to use as the source of the 'nuget' artifact: '', 'windows', 'macos', 'linux'
-  publishOutputSuffix: ''                                   # the artifact suffix to use when publishing the output folder
-  signListPath: 'SignList.xml'                              # the path to the SignList.xml to copy into the nuget artifact for signing
-  artifactsPath: 'output'                                   # the path to the NuGet packages that need to be signed, verified and published
+  artifactsPath: 'output'                                           # Path to the NuGet packages that need to be signed, verified and published
+  signListPath: 'SignList.xml'                                      # Path to 'SignList.xml' used for signing NuGet packages
 
 jobs:
-  - job: ${{ parameters.name }}
-    strategy:
-      matrix:
-        macos:
-          poolName: ${{ parameters.macosAgentPoolName }}
-          imageName: ${{ parameters.macosImage }}
-          classicInstallerUrl: ${{ parameters.classicXAPkg }}
-          runCodeQL: false
-        windows:
-          poolName: ${{ parameters.windowsAgentPoolName }}
-          imageName: ${{ parameters.windowsImage }}
-          classicInstallerUrl: ${{ parameters.classicXAVsix }}
-          runCodeQL: true
-          runAPIScan: true
-    displayName: Build
-    timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
-    variables:
-      Codeql.Enabled: $(runCodeQL)
-    pool:
-      name: $(poolName)
-      vmImage: $(imageName)
-
-    steps:
-      - template: setup-environment.yml
-        parameters:
-          dotnetVersion: ${{ parameters.dotnetVersion }}
-          dotnetWorkloadRollbackFile: ${{ parameters.dotnetWorkloadRollbackFile }}
-          dotnetWorkloadSource: ${{ parameters.dotnetWorkloadSource }}
-          dotnetNuGetOrgSource: ${{ parameters.dotnetNuGetOrgSource }}
-          dotnetTools: ${{ parameters.tools }}
-          classicInstallerUrl: $(classicInstallerUrl)
+- job: build_${{ parameters.name }}
+  displayName: ${{ parameters.name }}
+  timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+  pool: ${{ parameters.buildPool }}
+  
+  templateContext:
+    sdl:
```

Comment:
Another parameter you can set on the `sdl` element to potentially speed things up here is:

```yaml
binskim:
  scanOutputDirectoryOnly: true
```